src/App.tsx

```diff
@@ -1,61 +1,111 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './Types/Types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [actualTodos, setActualTodos] = useState([...todos]);
+  const [actualInput, setActualInput] = useState('');
+  const [actualOption, setActualOption] = useState(0);
+  const [inputError, setInputError] = useState(false);
+  const [optionError, setOptionError] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isInputError, setIsInputError] = useState(false);
  const [isOptionError, setIsOptionError] = useState(false);
```